gui/__init__.py

```diff
@@ -0,0 +1 @@
+from .gui import MainDialog
```

Comment:
The gui module is broken up into multiple files but only the MainDialog is needed when running it.